apps/web/src/pages/CommunityInterests/sections/FindANewCommunity.tsx

```diff
@@ -0,0 +1,283 @@
+import { useIntl } from "react-intl";
+import UserGroupIcon from "@heroicons/react/24/outline/UserGroupIcon";
+import { useFormContext } from "react-hook-form";
+import { ComponentProps } from "react";
+
+import { Heading, Well } from "@gc-digital-talent/ui";
+import { Checklist, RadioGroup, Select } from "@gc-digital-talent/forms";
+import {
+  commonMessages,
+  errorMessages,
+  uiMessages,
+} from "@gc-digital-talent/i18n";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+
+import { FormValues } from "../form";
+
+export const FindANewCommunityOptions_Fragment = graphql(/* GraphQL */ `
+  fragment FindANewCommunityOptions_Fragment on Query {
+    communities {
+      id
+      name {
+        localized
+      }
+      workStreams {
+        id
+        name {
+          localized
+        }
+      }
+    }
+  }
+`);
+
+export interface SubformValues {
+  communityId: string | null;
+  jobInterest: string | null;
+  trainingInterest: string | null;
+  interestInWorkStreamIds: string[] | null;
+}
+
+interface FindANewCommunityProps {
+  optionsQuery: FragmentType<typeof FindANewCommunityOptions_Fragment>;
+  formDisabled: boolean;
+  mode: "create" | "update";
+}
+
+const FindANewCommunity = ({
+  optionsQuery,
+  formDisabled,
+  mode,
+}: FindANewCommunityProps) => {
+  const intl = useIntl();
+  const optionsData = getFragment(
+    FindANewCommunityOptions_Fragment,
+    optionsQuery,
+  );
+
+  const { watch } = useFormContext<FormValues>();
+  const [selectedCommunityId] = watch(["communityId"]);
+  const allForm = watch();
```

Comment:
`allForm` is assigned a value but never used    
```suggestion
```

Reply:
Good catch, I was using that for debugging and forgot it.